src/Microsoft.Health.Dicom.Core/Models/Operations/DurableOrchestrationInstanceState.cs

```diff
@@ -0,0 +1,32 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Dicom.Core.Models.Operations.Serialization;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Health.Dicom.Core.Models.Operations
+{
+    internal class DurableOrchestrationInstanceStatus
+    {
```

Comment:
Looks like the implementation on AzureFunction side is missed from this PR, would you like to add it so that can view it end to end?

Reply:
What do you?

Reply:
My bad, it's implemented as `DurableClientProxyFunctions`